content/docs/rfcs/17/README.md

```diff
@@ -110,6 +110,12 @@ If the `epoch` attached to the message is more than `max_epoch_gap` apart from t
 This is to prevent a newly registered peer from spamming the system by messaging for all the past epochs. 
 `max_epoch_gap` is a system parameter for which we provide some recommendations in section [Recommended System Parameters](#recommended-system-parameters).
 
+**Merkle Root Validation**
+The routing peers MUST check whether the provided merkle root in the RateLimitProof is valid.
+It can do so by maintaining a local set of valid merkle roots, which consist of `AcceptableRootWindowSize` past roots.
+This allows peers which are not well connected to the network to be able to send messages, accounting for propagation delay.
```

Comment:
Perhaps useful to be as precise as possible here on why "not well connected" could imply using an old merkle root.